app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -46,6 +46,10 @@ def search_by_job_invocation(key, operator, value)
       end
     end
 
+    def cockpit_url
+      SSHExecutionProvider.cockpit_url_for_host(self.name)
```

Comment:
This was recently renamed, but we did not update all usages anyway and in order for this to be cherry-pickable, I'm happy we are using the old name here :+1: